src/Shared/ServerCertificateValidationProvider.cs

```diff
@@ -45,7 +45,12 @@ private static bool LoadCertificateToTrustedCollection(X509Certificate2Collectio
     {
         try
         {
+#if NET10_0_OR_GREATER
+            collection.Add(X509CertificateLoader.LoadCertificateFromFile(certFileName));
```

Comment:
Resolves [SYSLIB0057: X509Certificate2 and X509Certificate constructors for binary and file content are obsolete](https://learn.microsoft.com/dotnet/fundamentals/syslib-diagnostics/syslib0057)